src/components/v5/common/ActionSidebar/partials/DefaultField/DefaultField.tsx

```diff
@@ -39,7 +39,7 @@ const DefaultField: FC<DefaultFieldProps> = ({
       />
       {isDirty && isError && (
         <div
-          className={clsx('text-xs shrink-0 ml-auto', {
+          className={clsx('text-xs shrink-0 ml-auto absolute right-0 top-5', {
```

Comment:
You should also add a padding to the input, so that the value and this counter won't overlap